http-client/mhs/asynchronous-express-message-pattern/Return-information-from-ebxml-fault-returned-from-spine-in-original-post-request-when-sync-async-requested.http

```diff
@@ -0,0 +1,16 @@
+
+
+@PATIENT_NHS_NUMBER = 9689177923
+@DISSENTOVERRIDE = dissentOverride
+# @name Return_information_from_ebxml_fault_returned_from_spine_in_original_post_request_when_sync_async_requested
+POST http://localhost HTTP/1.1
```

Comment:
Host name should be a variable. It might be needed to create an environment file that points to exemplar on AWS which will have different hostname